### PR TITLE
Some additions / Fixes / Testing

### PR DESCRIPTION
**Pull request by loganwright**

### CODE REVIEW

File: SwiftWisdom/Core/Colors/ColorDescriptor.swift [hunk 1]
```diff
@@ -64,18 +79,25 @@ extension ColorDescriptor : StringLiteralConvertible, RawRepresentable, Equatabl
         self.init(rawValue)
     }
     
-    init(_ string: String) {
+    public init(_ string: String) {
         let rgbComponents = string.componentsSeparatedByString(",")
-            .flatMap { Int($0) }
         if rgbComponents.count == 4 {
-            self = .RGB(r: rgbComponents[0], g: rgbComponents[1], b: rgbComponents[2], a: rgbComponents[3])
+            if rgbComponents.first?.containsString(".") == true {
```
Comment (paulrolfe) [inline SwiftWisdom/Core/Colors/ColorDescriptor.swift]: What if it's just `0`? i.e. rgb values of `0, 0.8, 0.4, 1`

File: SwiftWisdom/Core/UIKit/UIImage+Overlay.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    public final func ip_overlayImage(overlayImage: UIImage, inRect innerRect: CGRect) -> UIImage {
+        let outerRect = CGRect(origin: CGPointZero, size: size)
+        UIGraphicsBeginImageContext(outerRect.size)
+        self.drawInRect(outerRect)
+        overlayImage.drawInRect(innerRect)
+        let img = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+        return img
+    }
+}
```
Comment (paulrolfe) [inline SwiftWisdom/Core/UIKit/UIImage+Overlay.swift]: new line at end.

File: SwiftWisdom/Core/Colors/ColorDescriptor.swift (hunk 1 above)
Comment (loganwright) [inline SwiftWisdom/Core/Colors/ColorDescriptor.swift]: The alternative I was toying with is checking the entire string for a `.`, that might be a better approach.

Comment (paulrolfe) [inline SwiftWisdom/Core/Colors/ColorDescriptor.swift]: Yeah, I think that's a good idea.

**Comment by paulrolfe** [comment]
:fire: :arrow_up:

File: SwiftWisdom/Core/StandardLibrary/Numbers/Int+Extensions.swift [hunk 3]
```diff
@@ -18,8 +18,9 @@ extension Int {
     }
 }
 
-extension Int {
+extension IntegerType {
     public func times(@noescape closure: Block) {
```
Comment (paulrolfe) [inline SwiftWisdom/Core/StandardLibrary/Numbers/Int+Extensions.swift]: Should this be `ip_times` to avoid ambiguity?

Comment (loganwright) [inline SwiftWisdom/Core/StandardLibrary/Numbers/Int+Extensions.swift]: yup, good call

**Pull request merged**